core/src/main/java/org/mule/runtime/core/privileged/processor/objectfactory/MessageProcessorChainObjectFactory.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.core.privileged.processor.objectfactory;
+
+import static java.lang.String.format;
+import org.mule.runtime.core.api.MuleContext;
+import org.mule.runtime.core.api.processor.MessageProcessorBuilder;
+import org.mule.runtime.core.api.processor.MessageProcessorChain;
+import org.mule.runtime.core.api.processor.MessageProcessorChainBuilder;
+import org.mule.runtime.core.api.processor.Processor;
+import org.mule.runtime.core.internal.processor.chain.ExplicitMessageProcessorChainBuilder;
+import org.mule.runtime.dsl.api.component.AbstractAnnotatedObjectFactory;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+public class MessageProcessorChainObjectFactory extends AbstractAnnotatedObjectFactory {
+
+  @Inject
+  protected MuleContext muleContext;
+  protected List processors;
+  protected String name;
+
+
+  public void setMessageProcessors(List processors) {
+    this.processors = processors;
+  }
+
+  @Override
+  public Object doGetObject() throws Exception {
```

Comment:
I removed the set annotations part that I saw repetead. Not sure what else do you saw repeated.